src/alert/alert.ts

```diff
@@ -13,8 +16,19 @@ import {Component, Input, Output, EventEmitter, ChangeDetectionStrategy} from '@
     `
 })
 export class NgbAlert {
+  /**
+   * A flag indicating if a given alert can be dismissed (closed) by a user. If this is flag is set a close button (in a
```

Comment:
I would say "If this flag is set, a close button..." There are two things to fix there, an extra "is" and the comma.